README.md

```diff
@@ -159,9 +159,20 @@ The Equinix Metal CCM works in one facility at a time. You can control which fac
 The overrides of environment variable and config file are provided so that you can run the CCM
 on a node in a different facility, or even outside of Equinix Metal entirely.
 
-### BGP
+### Load Balancers
+
+Equinix Metal does not offer managed load balancers like [AWS ELB](https://aws.amazon.com/elasticloadbalancing/)
+or [GCP Load Balancers](https://cloud.google.com/load-balancing/). Instead, Equinix Metal CCM is able to interface
+with and configure external bare-metal loadbalancers.
+
+Depending on the type of loadbalancer configured, the CCM may set Kubernetes `Node` annotations, enable BGP on the project
+and specific devices, getting an Equinix Metal Elastic IP, set the `Spec.LoadBalancerIP` on the Kubernetes `Service`, and
+other activities. Each loadbalancer implementation may have unique ways of working.
```

Comment:
The text that follows in `#### BGP` starts us down a path of mental branching. `If the loadbalancer implementation requires`, `Each loadbalancer implementation`.

I wonder if we are moving into generic text too soon and we could settle with being specific for now:
`MetalLB requires BGP...` `Kube-VIP requires an EIP...`.

I also wonder if a markdown table (right here) might help. We _might_ be able to represent the supported loadbalancers, their various modes, capabilities, and requirements in table-format.

Reply:
We might also benefit from keeping the kube-vip and metallb docs separated, as unique markdown files.

Reply:
I like the idea of separate markdown files quite a bit. I wasn't sure how to handle the bgp part, as it is a key part of design - CCM decides how to do bgp if needed - but specific to each. Hence the mental branching.

I will see if we can get some cleanup done on this, but also continue down this implementation path.

Reply:
@displague I cleaned it up somewhat. I didn't yet break it out into separate files, since I haven't included kube-vip in this yet, but rather set up the ability to do it. Take another look please.